packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/DynamicFromMap.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.bridge
+
+import androidx.core.util.Pools.SimplePool
+
+/** Implementation of Dynamic wrapping a ReadableMap. */
+internal class DynamicFromMap
+// This is a pools object. Hide the constructor.
+private constructor() : Dynamic {
+    private var mMap: ReadableMap? = null
+    private var mName: String? = null
```

Comment:
Can you update these to avoid using Hungarian notation?

```suggestion
    private var map: ReadableMap? = null
    private var name: String? = null
```

Reply:
I forgot about this requirement - sure, I will apply that.